legal-copy/legal-templates.js

```diff
@@ -1,18 +1,16 @@
-module.exports.templateBegin = `
-<!doctype html>
+module.exports.templateBegin = `<!doctype html>
 <html>
 <head>
-    <title>Test Pilot</title>
+    <meta charset="utf-8">
+    <title>Firefox Test Pilot</title>
```

Comment:
A couple of tweaks to make the legal templates more consistent with the index.html.